modules/sdk-evm-rpc-wrapper/test/unit/evmRPCWrapper.ts

```diff
@@ -0,0 +1,91 @@
+import { TestBitGo } from '@bitgo/sdk-test';
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { EvmRPCWrapper } from '../../src/evmRPCWrapper';
+import { Wallet, common, ECDSAUtils, Keychains, Ecdsa, SignedMessage } from '@bitgo/sdk-core';
+import { EVMRPCRequest } from '../../src/types';
+import { Gteth } from '@bitgo/sdk-coin-eth';
+import * as sinon from 'sinon';
+import {
+  ethWalletData,
+  txRequestForMessageSigning,
+  reqId,
+  typedMessage,
+  txRequestForTypedDataSigning,
+} from '../fixtures/evmRPCWrapperFixtures';
+import nock = require('nock');
+import { SignTypedDataVersion } from '@metamask/eth-sig-util';
+
+describe('method testing', function () {
+  const bitgo = TestBitGo.decorate(BitGoAPI, { env: 'test' });
+  const coin = 'gteth';
+  let params: EVMRPCRequest;
+  let txHash;
+  const sandbox = sinon.createSandbox();
+  let bgUrl;
+  const walletPassphrase = 'bitGo';
+
+  let wallet;
+  let evmRPCWrapper;
+  let signTxRequestForMessage;
+
+  before(function () {
+    bitgo.initializeTestVars();
+    bitgo.safeRegister(coin, Gteth.createInstance);
+    wallet = new Wallet(bitgo, bitgo.coin(coin), ethWalletData);
+    bgUrl = common.Environments[bitgo.getEnv()].uri;
+    evmRPCWrapper = new EvmRPCWrapper(wallet);
+    sandbox
+      .stub(Keychains.prototype, 'getKeysForSigning')
+      .resolves([{ commonKeychain: 'test', id: '', pub: '', type: 'independent' }]);
+    sinon.stub(Ecdsa.prototype, 'verify').resolves(true);
+    signTxRequestForMessage = sandbox.stub(ECDSAUtils.EcdsaUtils.prototype, 'signTxRequestForMessage');
+    nock.cleanAll();
+  });
+
+  it('personal_sign should call wallet signMessage', async function () {
+    signTxRequestForMessage.resolves(txRequestForMessageSigning);
+    const messageRaw = 'test message';
+    params = {
+      method: 'personal_sign',
+      params: [messageRaw],
+    };
+    const expected: SignedMessage = { txRequestId: reqId.toString(), txHash, messageRaw, coin };
+    const signStub = sandbox.stub(Wallet.prototype, 'signMessage').resolves(expected);
+    nock(bgUrl)
```

Comment:
Its always a good practice to make sure that at the end of the test, the nock was actually utilised, Check [this](https://github.com/nock/nock#isdone). This will make sure that nocks won't leak out to other tests and you could avoid using `nock.cleanAll();`